test/jasmine-node/upload/binding-spec.js

```diff
@@ -0,0 +1,169 @@
+var formData = require('form-data');
+
+var server = require('../server.js');
+var chain = require('../chain.js');
+var data = require('../data.js');
+var floats = require('../floats.js');
+
+/** @const */
+var bindingSpec = {};
+
+/**
+ * Data information for tests
+ * @type {{
+ *   name: string,
+ *   description: string,
+ *   fileName: string
+ * }}
+ */
+bindingSpec.dataInfo = {
+  name: 'wig-1',
+  description: 'the first binding track',
+  fileName: 'wig-1.bw'
+};
+
+/**
+ * @typedef {{
+ *   gene: string,
+ *   chr: string,
+ *   xMin: number,
+ *   xMax: number,
+ *   valueMin: number,
+ *   valueMax: number,
+ *   allValueMin: number,
+ *   allValueMax: number,
+ *   values: !Array<{x: number, value: number}>
+ * }}
+ */
+bindingSpec.QueryResponse;
+
+/**
+ * Test cases for binding queries.
+ * @type {!Array<{
+ *   name: string,
+ *   action: function(!frisby),
+ *   check: Function
+ * }>}
+ */
+bindingSpec.tests = [
+  // TODO(jiaming): Handle bigWigToWig processing time. The server needs to
```

Comment:
I did not handle this in the server. I am going to handle this in the next pull req with other issues. I think asynchronous function maybe better because it take about 3 minutes to upload the sample binding file.